ui/app/styles/components/info-table-row.scss

```diff
@@ -42,6 +42,10 @@
   .icon-false {
     color: $ui-gray-300;
   }
+
+  a {
```

Comment:
Do we want _all_ links to have this styling? If not we should add a class to the array links

Reply:
Yeah, I checked with Ivana on this.  I think it's another situation where --I think-- the design system is ahead of our code base.  Ivana mentioned that we don't underline links, but having to add this contradicts that.